l1-contracts/contracts/state-transition/verifiers/ZKsyncOSDualVerifier.sol

```diff
@@ -89,91 +80,24 @@ contract ZKsyncOSDualVerifier is IVerifier {
             revert UnknownVerifierVersion();
         }
 
-        if (verifierType == FFLONK_VERIFICATION_TYPE) {
-            return fflonkVerifiers[verifierVersion].verify(_publicInputs, _extractProof(_proof));
-        } else if (verifierType == PLONK_VERIFICATION_TYPE) {
-            return plonkVerifiers[verifierVersion].verify(_publicInputs, _extractProof(_proof));
-        } else if (verifierType == ZKSYNC_OS_PLONK_VERIFICATION_TYPE) {
+        if (verifierType == ZKSYNC_OS_PLONK_VERIFICATION_TYPE) {
             uint256[] memory args = new uint256[](1);
             args[0] = computeZKSyncOSHash(_proof[1], _publicInputs);
 
             return plonkVerifiers[verifierVersion].verify(args, _extractZKSyncOSProof(_proof));
-        } else if (verifierType == ZKSYNC_OS_MOCK_VERIFICATION_TYPE) {
-            // just for safety - only allowing default anvil chain and sepolia testnet
-            if (block.chainid != 31337 && block.chainid != 11155111) {
-                revert UnsupportedChainIdForMockVerifier();
-            }
-
-            uint256[] memory args = new uint256[](1);
-            args[0] = computeZKSyncOSHash(_proof[1], _publicInputs);
-
-            return mockverify(args, _extractZKSyncOSProof(_proof));
         }
         // If the verifier type is unknown, revert with an error.
         else {
             revert UnknownVerifierType();
         }
     }
 
-    function mockverify(uint256[] memory _publicInputs, uint256[] memory _proof) public view virtual returns (bool) {
-        if (_proof.length != 2) {
-            revert InvalidMockProofLength();
-        }
-        if (_proof[0] != 13) {
-            revert InvalidProof();
-        }
-        if (_proof[1] != _publicInputs[0]) {
-            revert InvalidProof();
-        }
-        return true;
-    }
-
     /// @inheritdoc IVerifier
     /// @dev Used for backward compatibility with older Verifier implementation. Returns PLONK verification key hash.
     function verificationKeyHash() external view returns (bytes32) {
         return plonkVerifiers[0].verificationKeyHash();
     }
 
-    /// @notice Calculates a keccak256 hash of the runtime loaded verification keys from the selected verifier.
-    /// @return The keccak256 hash of the loaded verification keys based on the verifier.
-    function verificationKeyHash(uint256 _verifierType) external view returns (bytes32) {
```

Comment:
are you sure it is okay to delete this variable? a similar function was used in Era, IDK how the things are going in zksync os